src/actions/LoginActions.tsx

```diff
@@ -222,12 +222,12 @@ export function initializeAccount(
             const { userSettings = {} } = currencyConfig
             currencyConfig
               .changeUserSettings(userSettings)
-              .catch((error: unknown) => {
-                showError(error)
+              .catch((err: unknown) => {
```

Comment:
Why rename this? If we had to standardize on one name, I would prefer "error" over "err" or "e".